tt_metal/llrt/blackhole/bh_hal_idle_eth.cpp

```diff
@@ -70,19 +70,32 @@ HalCoreInfoType create_idle_eth_mem_map() {
     std::vector<std::vector<HalJitBuildConfig>> processor_classes(NumEthDispatchClasses);
     std::vector<HalJitBuildConfig> processor_types(1);
     for (std::uint8_t processor_class_idx = 0; processor_class_idx < NumEthDispatchClasses; processor_class_idx++) {
-        DeviceAddr fw_base, local_init;
+        DeviceAddr fw_base, local_init, fw_launch;
+        uint32_t fw_launch_value;
         switch (processor_class_idx) {
             case 0: {
```

Comment:
not necessarily in scope of this change...but why are we using 0, 1, 2 rather than names?

Reply:
no reason but these can be dm0, dm1